components/details/Requires.tsx

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
 import { Box, Link, Typography } from '@mui/material';
+import useGetLang from '@/hooks/useGetLang';
 
 export default function Requires({ data }: { data?: string }) {
+  const lang = useGetLang();
   if (!data || data === '0') {
     return null;
   }
 
   return (
-    <Box pb={[1, 1, 2, 2]} sx={{ fontSize: '14px' }}>
+    <Box pb={[1, 1, 1.5, 1.5]} sx={{ fontSize: '14px' }}>
       <Typography fontWeight="bold" fontSize={14} component="span">
-        Requires:{' '}
+        {lang === 'en' ? 'Requires' : '关联 EIP'}:{' '}
```

Comment:
This works, but I think not a perfect solution, imagine there will be more languages in the future. We might need to upgrade in the future.